main.inc.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
-Version: 0.1.3
+Version: 0.1.3_dev
```

Comment:
don't change the version

Reply:
I have changed it to make sure anybody uses it will clearly see that is it not an official release. But feel free to remove it.